templates/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+  <h1>Введите имя и электронную почту</h1>
+  <form action="/welcome" method="post">

```

Comment:
После этого атрибут `action` больше не нужен
```suggestion
  <form method="post">
```